docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md

```diff
@@ -52,12 +52,19 @@ In this step, you will configure the yaml file required for Windows event logs a
 
 Any custom fields can be tagged along with the data in this step.
 
-Once the details are filled in, click on the **Download YAML File** button to get the yaml file.
+#### Enable process metric collection (Optional)
```

Comment:
Copy over any changes here that were made in the Linux doc

Reply:
I think you are referring to [this](https://github.com/SumoLogic/sumologic-documentation/pull/3304/files#diff-55d6813f44f119335535210cfe052874036cdba7aa2b1581edbb04eafdf9dfceR68) section in linux doc. But this is applicable only to linux collection. And does not apply for windows and mac

Reply:
It appears to be identical ![image](https://github.com/SumoLogic/sumologic-documentation/assets/56411016/59df898f-7921-4394-a884-6a2440f68b2d)

Reply:
@sumoanema Yes correct that's the section. To confirm, do you mean @JV0812 needs to delete this section from docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md and docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md?

Reply:
@kimsauce This para is only applicable to Linux doc, where the other two doc does not support log collection. In this PR we are just adding the **Enable process metric collection** for all three docs. In addition, we have formatted a existing note for better user readability.

Reply:
I was referring to the entire **Enable process metric collection** section, which is nearly identical in all three docs. Here's the Linux and Windows ones. I was just saying whatever changes you make to Linux section needs to be applied to the others so they match.

## **Linux**
<img width="1007" alt="Screenshot 2023-11-10 at 4 34 46 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/1a716ee3-a18b-47a0-b69c-03eb6647db10">

## **Windows**
<img width="1040" alt="Screenshot 2023-11-10 at 4 35 19 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/84abe2fb-1296-47a5-b124-c2a5127d4ad8">

Reply:
Thanks @kimsauce, I have updated the macOS and Windows doc inline with the Linux doc.